sql/analyzer/validate_create_table.go

```diff
@@ -836,6 +778,7 @@ func removeInSchema(sch sql.Schema, colName, tableName string) sql.Schema {
 }
 
 // TODO: make this work for CREATE TABLE statements where there's a non-pk auto increment column
+// TODO: verify that the above is fixed
```

Comment:
if it's fresh in your head and easy we could just add it

Reply:
double checked, and there are existing tests for this case that pass
https://github.com/dolthub/go-mysql-server/blob/main/enginetest/queries/create_table_queries.go#L120